test/system/TR8.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/sh
+# Test for missing r8
+#
+
+
+# Check physics
+if [ -d "${CAM_ROOT}/components/cam" ]; then
```

Comment:
is this something where eventually we'll want to check for kind_phys within physics/ncar_ccpp (atmospheric_physics)?

Reply:
I would think we might.   Should I leave this here as a sample/placeholder?

Reply:
Yeah - I think leaving it is fine.